src/vs/workbench/services/editor/common/editorOpenWith.ts

```diff
@@ -49,7 +49,12 @@ export async function openEditorWith(
 		return;
 	}
 
-	const overrideToUse = typeof id === 'string' && allEditorOverrides.find(([_, entry]) => entry.id === id);
+	let overrideToUse;
```

Comment:
@rebornix Would we ever want to show the reopen with prompt if there is only one override?